packages/hoprd/hopr-admin/commands/addresses.ts

```diff
@@ -0,0 +1,41 @@
+import { AbstractCommand } from './abstractCommand'
+import type PeerId from 'peer-id'
+import { checkPeerIdInput, styleValue } from './utils'
+
+export default class Addresses extends AbstractCommand {
+  constructor() {
+    super()
+    this.hidden = true
+  }
+
+  public name() {
+    return 'addresses'
+  }
+
+  public help() {
+    return 'Get the known addresses of other nodes'
+  }
+
+  public async execute(log, query: string): Promise<void> {
+    if (!query) {
+      return log(`Invalid arguments. Expected 'addresses <peerId>'. Received '${query}'`)
+    }
+
+    let peerId: PeerId
+    try {
+      peerId = checkPeerIdInput(query)
+    } catch (err) {
+      return log(styleValue(err.message, 'failure'))
+    }
+
+    return log(
```

Comment:
needs to be implemented, currently using `node`

Reply:
looks like we are missing `getObservedAddresses` API - lets ignore this command for now so we can merge this PR, can u simply comment out the failing code?

Reply:
sorted out!